assignments/objects.js

```diff
@@ -18,28 +18,31 @@ const example = {
 }
 
 // Write your intern objects here:
-
+let interns = [{"id": 0, "name": "Mitzi,F", "email" : "mmelloy0@psu.edu", "gender": "F"},
+  {"id": 1, "name": "Kennan,M", "email" :"kdiben1@tinypic.com", "gender": "M", speak: "Hello"},
+  {"id": 2, "name": "Keven,M", "email" :"kmummery2@wikimedia.org", "gender": "M"},
+  {"id": 3, "name": "Gannie,M", "email" :"gmartinson3@illinois.edu", "gender": "M"},
+  {"id": 4, "name": "Antonietta,F", "email" :"adaine5@samsung.com", "gender": "F", multiplyNums:  (a, b) => a*b}];
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
-
 // Mitzi's name
-
+console.log(Object.values(interns[0])[1]);
 // Kennan's ID
-
+console.log(Object.values(interns[1])[0]);
 // Keven's email
-
+console.log(Object.values(interns[2])[2]);
 // Gannie's name
-
+console.log(Object.values(interns[3])[1]);
 // Antonietta's Gender
-
+console.log(Object.values(interns[4])[3]);
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
```

Comment:
to console.log(kennan.speak());  you have to create a speak function inside of kennan's object.  The way you did it also works, but the hint provided in the instructions above indicates that it wants you to create a speak() method.  This is how:

{
    'id': 2,
    'name': 'Kennan',
    'email': 'kdiben1@tinypic.com',
    'gender': 'M',
    speak: function () {
      return 'Hello, my name is ' + this.name + '!';
    },
  }

now you can simply call that function by invoking it. You invoke a method by adding the () after it.
console.log(interns[1].speak());